tests/core/test_validation_definition.py

```diff
@@ -327,78 +329,53 @@ class TestValidationDefinitionSerialization:
     suite_name = "my_suite"
     validation_definition_name = "my_validation"
 
+    @pytest.fixture
+    def context(self, in_memory_runtime_context: EphemeralDataContext) -> EphemeralDataContext:
+        return in_memory_runtime_context
+
     @pytest.fixture
     def validation_definition_data(
         self,
-        in_memory_runtime_context: EphemeralDataContext,
+        context: EphemeralDataContext,
     ) -> tuple[PandasDatasource, CSVAsset, BatchDefinition]:
-        context = in_memory_runtime_context
-
         ds = context.data_sources.add_pandas(self.ds_name)
         asset = ds.add_csv_asset(self.asset_name, "data.csv")
         batch_definition = asset.add_batch_definition(self.batch_definition_name)
 
         return ds, asset, batch_definition
 
     @pytest.fixture
-    def validation_definition_suite(self) -> ExpectationSuite:
-        return ExpectationSuite(self.suite_name)
+    def validation_definition_suite(self, context: EphemeralDataContext) -> ExpectationSuite:
+        return context.suites.add(ExpectationSuite(self.suite_name))
 
     @pytest.mark.unit
-    @pytest.mark.parametrize(
-        "ds_id, asset_id, batch_definition_id",
-        [
-            (
-                "9a88975e-6426-481e-8248-7ce90fad51c4",
-                "9b35aa4d-7f01-420d-9d45-b45658e60afd",
-                "782c4aaf-8d56-4d8f-9982-49821f4c86c2",
-            ),
```

Comment:
This test's parameterization no longer makes sense because we can't get to serialization without having all our requisite ids